pkg/utils/release/controller.go

```diff
@@ -0,0 +1,132 @@
+package release
+
+import (
+	"context"
+	"fmt"
+	kubeCl "github.com/redhat-appstudio/e2e-tests/pkg/apis/kubernetes"
+	"github.com/redhat-appstudio/release-service/api/v1alpha1"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	k8sErrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type SuiteController struct {
+	*kubeCl.K8sClient
+}
+
+func NewSuiteController(kube *kubeCl.K8sClient) (*SuiteController, error) {
+	return &SuiteController{
+		kube,
+	}, nil
+}
+
+// CreateApplicationSnapshot creates a new ApplicationSnapshot using the given parameters.
+func (s *SuiteController) CreateApplicationSnapshot(name, namespace, applicationName string, snapshotImages []v1alpha1.Image) (*v1alpha1.ApplicationSnapshot, error) {
```

Comment:
As part of https://issues.redhat.com/browse/HACBS-665 the package containing the ApplicationSnapshot will change. This is fine for now though (it will change like this week).

Reply:
I will tag you in the PR changing it so you can see the new imports

Reply:
thanks, 
same answer as above